lib/commands/push.ts

```diff
@@ -259,17 +289,35 @@ export default class PushCmd extends Command {
 		const buildTarget = await this.getBuildTarget(appOrDevice);
 		switch (buildTarget) {
 			case BuildTarget.Cloud:
+				const _ = await import('lodash');
```

Comment:
A fair amount of code is being added to this `case BuildTarget.Cloud` "branch". What about moving both branches of the `switch` statement to separate functions? So it might read something like:
```
		switch (buildTarget) {
			case BuildTarget.Cloud:
				await this.pushToCloud(...);
				break;
			case BuildTarget.Device:
				await this.pushToDevice(...);
				break;
			default:
				...
```
Smaller functions with well defined purposes are usually easier to make sense of, and test (say we would one day want to write a test for it...). And the parent function would be easier to read as well.